benchmark/desktop/bridge/tools_api.py

```
@@ -0,0 +1,159 @@
+            self._send(handler, 200, {"tool": {"name": tool_name}, "raw_input": json.dumps(value, ensure_ascii=False), **result})
+        except Exception as exc:
+            self._send(handler, 400, {"error": str(exc), "output": str(exc), "is_error": True})
+
+    @staticmethod
+    def _send(handler: BaseHTTPRequestHandler, status: int, payload: dict[str, Any]) -> None:
+        data = json.dumps(payload, ensure_ascii=False).encode("utf-8")
+        handler.send_response(status); handler.send_header("Content-Type", "application/json"); handler.send_header("Content-Length", str(len(data))); handler.end_headers(); handler.wfile.write(data)
+
+
+def _read_body(handler: BaseHTTPRequestHandler) -> dict[str, Any]:
+    try: length = int(handler.headers.get("Content-Length", "0") or "0")
```

Ruff (0.16.2)

[error] Multiple statements on one line (colon)

(E701)